reference/forms/types/options/extra_fields_message.rst.inc

```diff
@@ -7,3 +7,9 @@ This is the validation error message that's used if the submitted form data
 contains one or more fields that are not part of the form definition. The
 placeholder ``{{ extra_fields }}`` can be used to display a comma separated
 list of the submitted extra field names.
+
+This message can be pluralized, see :ref:`formatting pluralized messages <component-translation-pluralization>` for details.
+
+    .. versionadded:: 5.1
```

Comment:
This should be moved line 4 just after the title and a blank line and before the type.

Reply:
Done.